### PR TITLE
Omit null

### PR DESCRIPTION
**Pull request by janvernieuwe**
~~DONT MERGE YET~~

To merge after zend-code has released a new version.
Update composer when this is done.

### CODE REVIEW

File: src/Phpro/SoapClient/CodeGenerator/Assembler/PropertyAssembler.php [hunk 1]
```diff
@@ -51,21 +51,20 @@ public function assemble(ContextInterface $context)
...
+                    'tags' => [
+                        [
+                            'name'        => 'var',
+                            'description' => $property->getType(),
+                        ],
+                    ]
                 ])
-            );
+            ]);
+            $propertyGenerator->omitDefaultValue();
```
Comment (veewee) [inline src/Phpro/SoapClient/CodeGenerator/Assembler/PropertyAssembler.php]: Isn't it possible to omit the default value with the `fromArray()` method?

Comment (janvernieuwe) [reply src/Phpro/SoapClient/CodeGenerator/Assembler/PropertyAssembler.php]: Not yet, might add it.

**Comment by janvernieuwe** [comment]
This can be merged, required php version needs to be upped to 7.1 though.

**Comment by veewee** [comment]
Hi @janvernieuwe,

Can we talk about PHP versions later today? Give me a ping when you're ready for it ;)

**Comment by janvernieuwe** [comment]
Just checked it, and the fromArray doesn't raise an error when the option is not defined.

File: test/PhproTest/SoapClient/Unit/CodeGenerator/Assembler/PropertyAssemblerTest.php [hunk 2]
```diff
@@ -48,7 +48,8 @@ function it_assembles_a_property()
         $assembler->assemble($context);
 
         $code = $context->getClass()->generate();
-        $expected = <<<CODE
+        if (version_compare(PHP_VERSION, '7.1', '>=')) {
```
Comment (veewee) [inline test/PhproTest/SoapClient/Unit/CodeGenerator/Assembler/PropertyAssemblerTest.php]: It's a better idea to create 2 separate tests and skip one of them based on the PHP version number.

**Pull request merged**